coding-concepts.js

```diff
@@ -1,73 +1,95 @@
-
+// // --------------------2) What will this log?
 const greeter = (name) => {
-  `Hello, ${name}!`
+   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student")) 
+
+// // a) Your answer:
+// This will not log anything. 
 
-// a) Your answer:
-// b) Verify and explain:
+// // b) Verify and explain:
+// This did not log anything, because there is no return statement on line 27. If there were a return statement, the output would read 'Hello, LEARN Student!'
 
 
-// --------------------3) What will this log?
+// // --------------------3) What will this log?
 
 var multBy2 = [4, 5, 6, 7, 8].map(value => value * 2)
-// console.log(multBy2)
+ console.log(multBy2)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:
+// This will log [8,10,12,14,16]
+// // b) Verify and explain:
+// This does log [8,10,12,14,16] because the .map method returns an array with the same amount of numbers, and value * 2 multiplies every number by 2. 
 
 
-// --------------------4) What will this log?
+// // --------------------4) What will this log?
 
 var onlyOdds = [11, 12, 13, 14, 15].filter(value => value % 2 !== 0)
-// console.log(onlyOdds)
+ console.log(onlyOdds)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:
+// This will log [11, 13, 15]
+// // b) Verify and explain:
+// This logs [11, 13, 15] because the .filter method filters out numbers based on a given condition, and the given condition is that odd numbers are filtered. This is why we see only odd numbers in the returned array. Value % 2 !== 0 means we are looking for numbers that are odd or rather, numbers that are not evenly divisible by 2. 
 
 
-// --------------------5) What will this log?
+// // --------------------5) What will this log?
 
 var myCodingSkills = {
   languages: ["JavaScript", "Ruby"],
   frameworks: ["React", "Ruby on Rails"],
   databases: "PostgreSQL",
   versionControl: "GitHub"
 }
-// console.log(myCodingSkills.languages[0])
+console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:
+// This will log JavaScript
+// // b) Verify and explain:
+//This logs JavaScript because console.log is referencing the index of 0 in the languages array, which is the JavaScript value. If JavaScript were the only word, this would return J. 
 
 
-// --------------------STRETCH: What will this log?
+// // --------------------STRETCH: What will this log?
 
-class Learn {
-  constructor(name){
-    this.student = name,
-    this.cohort = "Bravo",
-    this.year = 2022
-  }
-}
-var learnStudent = new Learn("George")
+// class Learn {
+//   constructor(name){
+//     this.student = name,
+//     this.cohort = "Bravo",
+//     this.year = 2022
+//   }
+// }
+// var learnStudent = new Learn("George")
 // console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:
+// // b) Verify and explain:
```

Comment:
Would really like to see an attempt on stretch problems.  Did you uncomment and run it to see what it returned?